buildout_1.rst

```diff
@@ -16,9 +16,9 @@ Topics covered:
 
 .. only:: not presentation
 
-    `Buildout <https://pypi.python.org/pypi/zc.buildout>`_ composes your application for you, according to your rules.
+    `Buildout <https://pypi.python.org/pypi/zc.buildout>`_ configures your application for you, according to your rules.
 
-    To compose your application you must define the eggs you need, which version, what configuration files Buildout has to generate for you, what to download and compile, and so on.
+    To configure your application you must define the eggs you need, which version, what configuration files Buildout has to generate for you, what to download and compile, and so on.
     Buildout downloads the eggs you requested and resolves all dependencies. You might need five different eggs, but in the end, Buildout has to install 300 eggs, all with the correct version in order to resolve all the dependencies.
```

Comment:
Yes and no:  compilation (of nginx, etc) is definitely different than configuration.  There are parts of Zope itself that need to be compiled, too.  OTOH, even the classic `configure/make/make install` cycle compiles stuff.  Perhaps we should use `install` instead of either `compose` or `configure`?
Where I disagree is that telling the system where to find zeo, front/backend, etc in my opinion is just what configuration means.  `zope.conf` is a `conf` file, not a `comp` file ;)  Also, I have never seen `compose` used in this context.